features/step_definitions/app_steps.rb

```diff
@@ -133,3 +133,7 @@
     element.should have_content("Edit")
     element.find("a", :text => "Edit", :exact => true).click
 end
+
+And /I click a delete button/ do
+    find(".glyphicon-remove").click
```

Comment:
What if we have other delete buttons elsewhere in the site? We should make this something else like "And I delete my conversations" or something, and add a "delete-conversation" class to the conversation delete divs.